content/en/docs/reference/kube-scheduler/profiles.md

```diff
@@ -0,0 +1,174 @@
+---
+title: Kube-scheduler Profiles
```

Comment:
To avoid awkward capitalization, I'd write:
```suggestion
title: Scheduling Profiles in kube-scheduler
```
or
```suggestion
title: Scheduling Profiles
```

Reply:
Done